src/main/ruby/core/file.rb

```diff
@@ -1129,22 +1120,15 @@ def self.unlink(*paths)
   # file to the first two arguments. Returns the number
   # of file names in the argument list.
   #  #=> Integer
-  def self.utime(a_in, m_in, *paths)
-    a_in ||= Time.now
-    m_in ||= Time.now
-    FFI::MemoryPointer.new(POSIX::TimeVal, 2) do |ptr|
-      atime = POSIX::TimeVal.new ptr
-      mtime = POSIX::TimeVal.new ptr[1]
-      atime[:tv_sec] = a_in.to_i
-      atime[:tv_usec] = 0
-
-      mtime[:tv_sec] = m_in.to_i
-      mtime[:tv_usec] = 0
-
-      paths.each do |path|
-        n = POSIX.utimes(Rubinius::Type.coerce_to_path(path), ptr)
-        Errno.handle unless n == 0
-      end
+  def self.utime(atime, mtime, *paths)
+    atime ||= Time.now
+    mtime ||= Time.now
```

Comment:
should these be the same `Time.now` for the case where both are falsey?

Reply:
It probably should, indeed.

Reply:
Fixed.